### PR TITLE
Using new protobuf lib

### PR DESCRIPTION
**Pull request by vitaliihonta**
Replaced `python3-protobuf` (an outdated for of the official `protobuf` lib) with the official `protobuf` lib.
It only required re-compiling protobuf files.

There were integration tests here. Some of them doesn't work (**and didn't work before my changes**) due to usage of non-standard riak bucket types, but most of them succeed. 
There are also a couple of tests in our `aioriak` fork. They work as well.

I'll test it on our live services (more details in private as **this is a public repo**)

### CODE REVIEW

**Comment by le-r-on** [comment]
Can you briefly explain how did you approach this change (e.g. maybe you re-used some existing draft open-source PR), and how you tested it?

File: riak/pb/messages.py [hunk 1]
```diff
@@ -1,17 +1,3 @@
-# Copyright 2010-present Basho Technologies, Inc.
```
Comment (le-r-on) [inline riak/pb/messages.py]: I am not sure you can just delete the licence...

Comment (vitaliihonta) [reply riak/pb/messages.py]: Good gotcha! That's autogenerated code, and it removed the license for some reason. I'll bring it back

**Comment by vitaliihonta** [comment]
@le-r-on added a better description

**Review by oleksandra-martyntseva — APPROVED** [review]

**Pull request merged**